packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -0,0 +1,36 @@
+---
+id: Right-to-left handbook
+title: PatternFly right-to-left handbook
+section: developer-resources
+
+---
+
+To allow for internationalization of your product's content, it is important to implement bidirectional language support by developing for both right-to-left (RTL) and left-to-right (LTR) languages. 
+
+The default browser behavior displays content following LTR formatting. To support RTL languages, like Arabic and Hebrew, you will need to explicitly add RTL support. To support RTL languages the browser uses `dir="RTL"` and `lang="{language code}"` to lay out the page appropriately for the selected language. Setting `dir="RTL"` causes the browser to reverse and lay out the contents from right to left. The W3C provides [information about internationalization and the bidirectional algorithm](https://www.w3.org/International/techniques/authoring-html#direction).
+
+## Support in PatternFly
+
+PatternFly components have been written to use logical properties - `block-start`, `inline-end`, `block-end`, and `inline-start` - rather than `top`, `right`, `bottom`, and `left`. This allows styles like margin and padding to be correctly laid out in both LTR and RTL. However, to avoid a sweeping breaking change, PatternFly variable names still use `top`, `bottom`, `left`, and `right`.
+
+PatternFly components have been written to reverse directional icons that are part of the component - for example, the caret indicating expansion of various components is properly reversed. 
+
+Other icons you may have in your interface will not be mirrored automatically, as it may be appropriate to mirror some but not others. Be sure to review all icons in your product and adjust appropriately. PatternFly has modifier classes listed below that can help with that.
+<br />
+
+## PatternFly modifiers supporting RTL
```

Comment:
Not a blocker, but would this paragraph make more sense in this section with some tweaking?

```suggestion
<br />

## PatternFly modifiers supporting RTL

Some elements you have in your interface will not be adjusted for RTL by PatternFly automatically, or you may require more customized adjustment. The following PatternFly modifier classes can be used for more fine-tuned control.
```

Reply:
I like this phrasing!